modules/client-configuration/pages/clients-centos.adoc

```diff
@@ -139,16 +139,26 @@ For more information about how to resolve this error, see ``Registering Older Cl
 When you update {centos} clients, the packages do not include metadata about the updates.
 You can use a third-party errata service to obtain this information.
 
+[WARNING]
+====
+
 ifeval::[{suma-content} == true]
 
-[IMPORTANT]
-====
-The third-party errata service described here is provided and maintained by the community.
+The third-party errata service described here, CEFS, is provided and maintained by the community.
 It is not supported by {suse}.
-====
 
 endif::[]
 
+The authors of CEFS provide patches (errata) on a best-effort basis, in the hope they will be useful but with no guarantees of correctness or currency.
```

Comment:
@Loquacity: I took the liberty to change this line after your suggestions from "patches and updates" to "patches (errata)".

Rationale: CEFS does not provide updates per se, only metadata about updates, so "updates" does not seem to me to be the optimal word here. I could live with "patches" only, as it is understood / should be in our glossary that patches are pure metadata.

Reason why I mentioned errata is that "patch" is SUSE/Uyuni/Manager jargon and "errata" is Red Hat/Spacewalk/Satellite jargon for exactly the same concept. When browsing for information around CEFS or in CEFS script commandline help, "errata" is used exclusively, while we use "patch" everywhere in our docs. So I thought about adding a suggestion just in case the reader is not really aware they are synonyms.

Is this OK from your POV? Suggestions welcome!

Reply:
ah, ok. i really dislike parentheses, though. can we make it "patches or errata" ?

Reply:
Sure, fixed